certbot/client.py

```diff
@@ -163,7 +163,7 @@ def perform_registration(acme, config):
 
 
 class Client(object):
-    """ACME protocol client.
+    """Certbot's client.
```

Comment:
Probably. This class is still an ACME client in the sense that it uses the ACME protocol, however, it does so at a higher level and contains a lot of Certbot specific code. I'm fine with either docstring.